schemas/Clinical/index.js

```diff
@@ -346,7 +346,7 @@ const convertClinicalSubmissionDataToGql = (programShortName, data) => {
         ),
       );
     },
-    fileErrors: fileErrors.map(fileError => convertClinicalSubmissionFileErrorrToGql(fileError)),
+    fileErrors: fileErrors.map(convertClinicalSubmissionFileErrorrToGql),
```

Comment:
@hlminh2000  just for you :smile:

Reply:
lol I feel like that one FP guy in Jon's poster